mantidproject/mantid
# MPI Build functional on Windows

Description of work.

MPI Build of Mantid configured on windows following work done to include an experimental MPI build of Mantid. 

**To test:**

- Enable MPI_Experimental in CMAKE and build
- Ensure all unit tests, in particular `Parallel` pass.
 
Fixes #19274 .



---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
